tests/framework/stats/consumer.py

```diff
@@ -141,6 +141,12 @@ def process(self, fail_fast=False) -> (dict, dict):
                         self._failure_aggregator.add_row(fail_msg)
                         if fail_fast:
                             raise self._failure_aggregator
+                else:
+                    self._statistics[ms_name][st_def.name]["outcome"] = "FAILED"
+                    fail_msg = f"'{ms_name}/{st_def.name}': Criteria not found."
```

Comment:
Can we be more specific here? I.e. `Criteria not found for {instance} / {cpu} / {kv}`

Reply:
second that

Reply:
That might be helpful, but those information are outputted in other part like these:
- https://github.com/firecracker-microvm/firecracker/blob/main/tools/devtool#L1153
- https://github.com/firecracker-microvm/firecracker/blob/main/tools/devtool#L1193

Do you think these information in each line?

Reply:
I also do not think it makes sense to log them; these are actually input info, it would mean to output info that the test is depending on (input info).

Reply:
If the information is not at hand, let's leave it like this. But I think there's value on having the information consolidated in one place.

Reply:
OK. I think both options have a point.
Could you guys take a look at the following samples and tell me which one you prefer again?

Let's say, it lacks all baselines for block performance on m6i.metal / Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz / kernel 5.10.

Option 1: original
```
[Firecracker devtool] Kernel version: 5.10.147-133.644.amzn2.x86_64
// snipped
11:02:50 block_performance: INFO Testing on processor Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz
// snipped
11:02:54 block_performance: INFO Testing with microvm: "1vcpu_1024mb.json", kernel vmlinux-4.14.bin, disk ubuntu-18.04.ext4
// snipped
E   framework.stats.core.CoreException: Failed on 'vmlinux-4.14.bin/ubuntu-18.04.ext4/async_1vcpu_1024mb.json/randrw-bs4096':
E   
E   'cpu_utilization_vmm/Avg': Criteria not found.
E   
E   'cpu_utilization_vcpus_total/Avg': Criteria not found.
E
E   Failed on 'vmlinux-4.14.bin/ubuntu-18.04.ext4/async_1vcpu_1024mb.json/randread-bs4096':
E   
E   'cpu_utilization_vmm/Avg': Criteria not found.
E   
E   'cpu_utilization_vcpus_total/Avg': Criteria not found.
// continued...
```

Option 2: add additional info
```
[Firecracker devtool] Kernel version: 5.10.147-133.644.amzn2.x86_64
// snipped
11:12:17 block_performance: INFO Testing on processor Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz
// snipped
E   framework.stats.core.CoreException: Failed on 'vmlinux-4.14.bin/ubuntu-18.04.ext4/async_1vcpu_1024mb.json/randrw-bs4096':
E   
E   'cpu_utilization_vmm/Avg': Criteria not found for m6i.metal / Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz / kernel 5.10.
E   
E   'cpu_utilization_vcpus_total/Avg': Criteria not found for m6i.metal / Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz / kernel 5.10.
E   
E   Failed on 'vmlinux-4.14.bin/ubuntu-18.04.ext4/async_1vcpu_1024mb.json/randread-bs4096':
E   
E   'cpu_utilization_vmm/Avg': Criteria not found for m6i.metal / Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz / kernel 5.10.
E   
E   'cpu_utilization_vcpus_total/Avg': Criteria not found for m6i.metal / Intel(R) Xeon(R) Platinum 8375C CPU @ 2.90GHz / kernel 5.10.
// continued...
```